core/server/data/schema/schema.js

```diff
@@ -452,6 +452,8 @@ module.exports = {
         stats: {type: 'text', maxlength: 65535, nullable: true},
         email_count: {type: 'integer', nullable: false, unsigned: true, defaultTo: 0},
         subject: {type: 'string', maxlength: 300, nullable: true},
+        from: {type: 'string', maxlength: 191, nullable: true},
+        reply_to: {type: 'string', maxlength: 191, nullable: true},
```

Comment:
Corresponding value in `settings` table is keyed by `members_reply_address`, wondering if keeping the convention to `reply` or having in mind to fix settings in the future is a better way forward :thinking:

Reply:
Hmm. In this table `reply_to` more sense than `reply` because it matches the header name. I'd be in favour of renaming the settings value to `members_reply_to_address` if the only thing it's used for is setting the reply-to header on emails.

Reply:
Yup, renaming `members_reply_to_address` some time in the future would make more sense :+1: